modules/op-release-notes-1-8.adoc

```diff
@@ -44,8 +44,29 @@ Workaround: Use `nodejs:14-ubi8` instead of the `nodejs:14-ubi8-minimal` image s
 // (link:https://issues.redhat.com/browse/SRVKP-1782[#SRVKP-1782])
 // Writer, also see https://github.com/tektoncd/operator/blob/main/cmd/openshift/operator/kodata/tekton-addon/addons/02-clustertasks/source_local/s2i-nodejs/s2i-nodejs-task.yaml. Overkill: The user can run the `$ oc edit clustertask s2i-nodejs` command and replace `$(params.VERSION)` with `14-ubi8`. Alternatively, you can use a `TaskRun` resource to set the tag of the `nodejs` image stream to `14-ubi8`.
 
-// [id="fixed-issues-1-8_{context}"]
-// == Fixed issues
+[id="fixed-issues-1-8_{context}"]
+== Fixed issues
 
-// Pipelines
-// *
+* With this update, as a developer, you can add your Git repository to {pac} by using the webhook mechanism. This way, you can use {pac} even if GitHub Apps is not feasible.
+// (link:https://issues.redhat.com//browse/SRVKP-2137[SRVKP-2137 Pipeline as code - CLI add repository via webhooks])
+// Shivam Mukhade
+
+* With this update, as a developer, you can use the *Developer* perspective of the {product-title} web console to run all the `PipelineRun` resources in your `.tekton` directory that match specified events. The console displays the status and reports of the runs. If you are using WebVCS, the reports display comments.
```

Comment:
this is also a new feature

Pipelines As Code used to support running only one PipelineRun from `.tekton` directory from user git repository 
now with this feature you can have multiple pipelinerun in their CI 
for example
![image](https://user-images.githubusercontent.com/55777192/182505476-97b70954-ed15-417f-b8a7-fcc0fd82b3e6.png)

Reply:
Updated. Thanks!